sdk/servicebus/service-bus/README.md

```diff
@@ -21,11 +21,11 @@ Use the client library for Azure Service Bus in your Node.js application to
 
 Install the Azure Service Bus client library using npm
 
-`npm install @azure/service-bus`
+`npm install @azure/service-bus@dev`
```

Comment:
Nice, did not know that we have a dev tag.

But I think we have to do `2.0.0-preview.1` or `next`.

Reply:
Harsha is right
`dev` tag would get the nightly builds that we have.

Also, I want to discuss options to ensure that a user looking for the stable build doesnt end up using the samples here and is re-directed properly. My previous idea was to add the note that you see in line 12, but now I feel we should have something more prominent. And also maybe at the start of every H3 or H2 just cause people might sometimes scroll to the header they are interested and skip the top part.

Thoughts?

Reply:
Good thoughts (from both of you!).

- I can definitely reference preview.1 explicitly. We're just in that annoying state that none of the bits are "published" under a nice tag yet :)
- For the "user looking for samples" we have (AFAIK) two avenues they could find these:
  - Via docs.microsoft.com : @willmtemple recently removed the "front matter" that would cause docs.microsoft.com to index these samples. So at least there, only the stable samples will show up.
  - Via github: this one is problematic. @HarshaNalluru was thinking we could just make a separate folder so we could at least host them side-by-side. @willmtemple , have we adopted any sort of convention around this yet (preview and stable hosted side-by-side?)

Reply:
Ah.. Thanks for asking, this reminded me that I wanted to mention regarding samples-v1 here. 

For now, please keep the copy of samples for version-1 here in a separate folder `samples-v1` until we have the plan to manage both the versions.

Reply:
@HarshaNalluru  - I'm going to link to them via the git tag - is that not enough? Why do they need to be in master?

Reply:
After talking with @HarshaNalluru  I believe we're good with the approach we've got here.

Reply:
Okay, only keeping v2-preview-version, deleting the v1-samples is fine.

Reply:
@ramya-rao-a  - I've now inserted a comment into:

- Each source file (.js and .ts)
- The readmes for each folder

So hopefully that's enough of a breadcrumb :)